envoy/Makefile

```diff
@@ -40,28 +54,46 @@ all: clean-bins release
 else
 BAZEL_OPTS ?=
 BAZEL_BUILD_OPTS = --experimental_strict_action_env
-
-all: clean-bins envoy api
+all: clean-bins release
```

Comment:
Why this change?

Reply:
`release` already depended on `api`. This has the exact same effect, in the sense that the exact same targets are being depended on by `all`.

Reply:
You changed the build target, effectively from bazel "fastbuild" to "opt", you must have had a rationale for that?

Reply:
Oh, I want to use `opt` for Istio to minimize the sidecar container size.
It doesn't matter as much for the node proxy used for dev. I'll update that.

Reply:
`make` will build release (`opt` build) if PKG_BUILD is defined. docker image builds use that, maybe you could too, without changing the default make targets?